doc/source/releases/0.7.0.rst

```diff
@@ -34,7 +35,7 @@ Removed:
 
 (Full `list of closed issues`_)
 
-Thanks to Achilleas Koutsou, Andrew Davison, Bj�rn M�ller, Chadwick Boulay, erikli, Jeffrey Gill, Julia Sprenger, Lucas (lkoelman), 
+Thanks to Achilleas Koutsou, Andrew Davison, Björn Müller, Chadwick Boulay, erikli, Jeffrey Gill, Julia Sprenger, Lucas (lkoelman), 
```

Comment:
GitHub claims this line is still changed. I don't think there were whitespace changes, so I'm not sure why. It seems fine to me. Ready to merge, I believe.